docs/getting-started/summaryofchanges.md

```diff
@@ -56,6 +60,21 @@ The following features and enhancements were added.
 - Added a more helpful and actionable description to message ZWEAM511E, which occurs when API ML does not trust the certificate provided by the service. [#818](https://github.com/zowe/api-layer/issues/818)
 
 #### Zowe App Server
+- Added option to write JSON files without using Node.js. [#137](https://github.com/zowe/zlux-app-server/pull/137)
+- Register ZSS to Zowe API ML using a configuration script. This is similar to how the API ML team registers their APIs (for example, the Jobs API). [#208](https://github.com/zowe/zss/pull/208)
+- Additional environment variables are now supported, which provides more options for TN3270 for the `instance.env` configuration file [#1176](https://github.com/zowe/zowe-install-packaging/issues/1176) while also allowing TN3270 host to be specified during installation configuration. [#1125](https://github.com/zowe/zowe-install-packaging/issues/1125). The following new env vars are now supported in `instance.env` [#108](https://github.com/zowe/zlux-app-server/pull/108):
+
+  - `ZOWE_ZLUX_TELNET_HOST = string`
+  - `ZOWE_ZLUX_SSH_HOST = string`
+  - `ZOWE_ZLUX_TN3270_ROW = number`
+  - `ZOWE_ZLUX_TN3270_COL = number`
+  - `ZOWE_ZLUX_TN3270_MOD = numbers 2-5 as well as "dynamic" or other variations of the word`
+  - `ZOWE_ZLUX_TN3270_CODEPAGE = ccsid number or string as seen in the ui`
+- An Agent API that provides limited information without the need for authentication has been added. [#211](https://github.com/zowe/zss/pull/211)
```

Comment:
The /server/agent/environment API already existed, but used to be exclusively for admins as it contained system information. Now, it can be used by everyone and non-admins will see a subset of what admins will see, specifically only those attributes pertaining to the functionality of zowe (OS, architecture, and environment variables about zowe configuration such as components used and ports they are accessible on)